File: string_transformer.py
def string_transformer(s):
    word_lst = list(s)
    change_wrd = ''.join([x.upper() if x.islower() is True else x.lower() if x.isupper() is True else x for x in word_lst]).split(' ')
    change_wrd.reverse()
    return ' '.join(change_wrd)


print(string_transformer('Example string'))# 'STRING eXAMPLE'

File: stringTransformer.js
function stringTransformer(str) {
    let upp = /[A-Z]/
    let low = /[a-z]/
    let results = [];
    let strRR = str.split('')
    for(let char in strRR){
      current = str[char]
      if(upp.test(current)){
        results.push(current.toLowerCase())
      }
      if(low.test(current)){
        results.push(current.toUpperCase())
      }
      if(current === ' '){
        results.push(current)
      }
    }
    return results.join('').split(' ').reverse().join(' ')
  }

console.log(stringTransformer('Example string'))//--> 'STRING eXAMPLE'